docs/use.html

```diff
@@ -0,0 +1,722 @@
+  </tr>
+</table>
+<p>Example:</p>
+<pre><code>@page styles Styles
+@group styles.colors 0 Colors
+@group styles.typography 1 Typography
+@group styles.forms-elements 2 Form Elements
+
+
+This is my style guide.
+
+</code></pre>
+<p>This will create the groups: <strong>Colors</strong>, <strong>Typography</strong>, and <strong>Form Elements</strong> under the sidebar shown for the <strong>Styles</strong> section:</p>
+<p><img src="static/img/style-guide-tag-group.png"/></p>
+<h2>Creating a Page</h2>
+<p>Now that you have a grasp on what tags are used for, let's dive into how to create your Living Style Guide.</p>
+<p>The first step is to create a <strong>page</strong> which will be used as the index of your styleguide. To create this page you can create a file in the <code>styles</code> directory called <code>styles.md</code>, and inside add the tag <a href="use.html#section=section__page">@page</a>. Your code should look something like this:</p>
+<pre><code>@page styles Styles
+
+This is my style guide.
+
+</code></pre>
+<p>Anything after the line with the tag will be used as text on your page.</p>
+<h3>Using Markdown</h3>
+<p>With DocumentCSS you can use <a href="https://en.wikipedia.org/wiki/Markdown">markdown</a> to structure semantically your documention. Markdown is great becuase is a lightweight markup that won't clutter the comments inside of your style sheets. For example, if you want to call out a piece of text to be displayed as &quot;code&quot; in markdown you can simply wrap the text in &quot;`&quot;. For example:</p>
+<table>
+  <tr>
+      <th style="text-align: right">This Markdown:</th>
+      <td>&#96;code&#96;</td>
+  </tr>
+  <tr>  
+      <th style="text-align: right">Will display in the generated HTML as:</th>
+      <td>&lt;code&gt;code&lt;/code&gt;</td>
+  </tr>
+  <tr>
+      <th style="text-align: right">Will display on the page as:</th>
+      <td><code>code</code></td>
+  </tr>
+</table> 
+<p>Reference this <a href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet">markdown cheatsheet</a> created by <strong>gitbub</strong> to learn more about how to use markdown in your docs.</p>
+<h3>Hyperlinking</h3>
+<p>To create links within your documentation simply use the <code>NAME</code> of the page or stylesheet that has been documented using the tags <code>@page</code> or <code>@stylesheet</code> and wrap it in <code>[]</code>. For example:</p>
+<p>The following page</p>
+<pre><code>@page accesibility Accesibility
+
+</code></pre>
+<p>Can be hyperlinked like this:</p>
+<pre><code>Referece [accesibility] for more details.
+
+</code></pre>
+<p>For links external to your style guide, use <a href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links">markdown</a>.</p>
+<h3>Adding images</h3>
+<p>From time to time you may want to add an image to your style guide. For this use as well <a href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images">markdown</a>.</p>
```

Comment:
This reads a little weird. Maybe use: "You can use regular markdown to add images to your style guide."